services/common/src/redux/slices/permitServiceSlice.ts

```diff
@@ -0,0 +1,203 @@
+import { createAppSlice, rejectHandler } from "@mds/common/redux/createAppSlice";
```

Comment:
@taraepp Put in actual implementations here for the 
- Fetch Status
- Delete permit conditions
actions